ancpbids/plugins/plugin_dsvalidator.py

```diff
@@ -5,6 +5,7 @@ class StaticStructureValidationPlugin(ValidationPlugin):
     def execute(self, dataset, report: ValidationPlugin.ValidationReport):
         self.schema = dataset.get_schema()
         gen = dataset.to_generator()
+        f = "./dataset_description.json"
```

Comment:
this is a hard-coded assumption that should be avoided, but I guess you just wanted to get started with an initial value?

Reply:
Of course — the comment lower down has a `#TODO ` that refers to solving this problem. It was a patch to make a test pass so that I could move on.